lib/engine/game/g_18_usa/game.rb

```diff
@@ -630,20 +630,76 @@ def operating_round(round_num)
             G1817::Step::CashCrisis,
             G18USA::Step::ObsoleteTrain,
             G18USA::Step::Loan,
+            G18USA::Step::ScrapTrain,
             G18USA::Step::SpecialTrack,
             G18USA::Step::SpecialToken,
             G18USA::Step::SpecialBuyTrain,
             G18USA::Step::Assign,
             G18USA::Step::Track,
             G18USA::Step::DenverTrack,
             G18USA::Step::Token,
+            G18USA::Step::BuyPullman,
             G18USA::Step::Route,
             G18USA::Step::Dividend,
-            Engine::Step::DiscardTrain,
+            G18USA::Step::DiscardTrain,
```

Comment:
The OR discard train step will need to be near the top of the list. There is a private that allows a company to buy a train anytime on their turn and that will need to trigger this step.

Reply:
I moved it near ScrapTrain